### PR TITLE
Nov 2016 meeting

### PR DESCRIPTION
**Pull request by h-lame**
We've got the registration page, and a couple of talks so this is enough to publish.

### CODE REVIEW

**Comment by tomstuart** [comment]
👍

**Comment by h-lame** [comment]
No typos, no wonky links?  I don't believe it.

File: source/meetings/2016/november/index.html.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+### AWS Elastic Beanstalk & Docker for Rails developers.
+
+[Tomas Valent](https://twitter.com/equivalent8) says:
+
+> Elastic Beanstalk (EB) is a product from AWS that is trying to provide easy
+> to setup load balanced environment of EC2 instances.
+>
+> In this talk we will configure simple Rails app, wrap it in a Docker container
+> and ship it to AWS EB setup. We will also play around with some configuration
+> options in order to squeeze maximum out of it.
```
Comment (tomstuart) [inline source/meetings/2016/november/index.html.md]: Maximum what?

File: source/meetings/2016/november/index.html.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+category: meeting
+title: November 2016 Meeting
+updated_at:
+published_at: 2016-10-17 13:20:16 +0100
+created_at: 2016-10-17 12:54:32 +0100
+parts: {}
+status: Published
+---
+
+The November 2016 meeting of LRUG will be on *__Tuesday__ the 15th of November*, from
```
Comment (tomstuart) [inline source/meetings/2016/november/index.html.md]: Might it be worth drawing even more attention to the unconventional day? e.g. “(NOTE UNCONVENTIONAL DAY)”

Comment (h-lame) [inline source/meetings/2016/november/index.html.md]: You're probably right; based on confusion folk had about October being 45 minutes later than usual the day being bold won't be enough.  I'll call it out more, and make sure to do the same in the email and tweet announcements.

File: source/meetings/2016/november/index.html.md (hunk 1 above)
Comment (h-lame) [inline source/meetings/2016/november/index.html.md]: I think it's maximum "I just paste in what the speaker said verbatim", I dunno. ("performance" maybe?)

**Pull request merged**